miden-lib/asm/sat/kernel.masm

```diff
@@ -233,6 +243,9 @@ end
 #!   - If ASSET is a fungible asset, then ASSET' is the total fungible asset in the account vault
 #!     after ASSET was added to it.
 export.account_vault_add_asset
+    # emit event to signal that an asset is being added to the account vault
+    push.1 drop emit.ADD_ASSET_TO_ACCOUNT_VAULT_EVENT
```

Comment:
I would add a `TODO` comment (with link to the issue) explaining why we do `push.1 drop`.